.github/workflows/reusable-examples.yml

```
@@ -808,6 +808,48 @@ jobs:
         working-directory: examples/nextjs-server-actions
         run: npm run build
 
+  nextjs-trpc:
+    name: Next.js + TRPC
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      # Environment security
+      - name: Harden Runner
+        uses: step-security/harden-runner@cb605e52c26070c328afc4562f0b4ada7618a84e # v2.10.4
+        with:
+          disable-sudo: true
+          egress-policy: block
+          allowed-endpoints: >
+            fonts.googleapis.com:443
+            fonts.gstatic.com:443
+            github.com:443
+            registry.npmjs.org:443
```

```suggestion
            registry.npmjs.org:443
            decide.arcjet.com:443
```

it's unclear why Next.js is calling Arcjet endpoints during build but this is just what we have to allow for now.